lib/system.nim

```diff
@@ -2106,7 +2106,7 @@ const
     ## is the minor number of Nim's version.
     ## Odd for devel, even for releases.
 
-  NimPatch* {.intdefine.}: int = 1
+  NimPatch* {.intdefine.}: int = 3
```

Comment:
What the heck? That should not be part of this PR!

Reply:
that was so code could branch on this bugfix, as explained in https://github.com/nim-lang/Nim/pull/14124#issuecomment-619520051 and it was useful in some PR's to branch on that.

It also makes `nim -v` a little more helpful to show, instead of just a hash + fixed 1.5.1 it shows hash + 1.5.x so it's immediately clearer in bug reports how recent was the nim version used.

Finally, it makes pre-releases / nightly builds easier to distinguish via `nim -v`

I'm not aware of negative consequences in bumping NimPatch, it's a common practice to bump patch on bug fixes. I'm not suggesting to do it on each commit (nim isn't following standard semver anyways), but bumping patch somewhat regularly between 2 stable releases is IMO a good thing.